dDatabase/CDClientManager.cpp

```diff
@@ -38,43 +38,53 @@
 #include "CDFeatureGatingTable.h"
 #include "CDRailActivatorComponent.h"
 
+// Uncomment this to cache the full cdclient database into memory. This will make the server load faster, but will use more memory.
+// A vanilla CDClient takes about 46MB of memory + the regular world data.
+// #define CDCLIENT_CACHE_ALL
```

Comment:
this could be a sharedconfig option

Reply:
well
that's a stupid comment
I meant cmake variable

Reply:
you should not need to rebuild the full server to define this imho

Reply:
added the variable uncommented for now